pkgs/development/interpreters/babashka/default.nix

```diff
@@ -54,8 +54,18 @@ buildGraalvmNativeImage rec {
       -s "https://api.github.com/repos/babashka/babashka/releases/latest" \
       | jq -r '.tag_name')"
 
-    # v0.6.2 -> 0.6.2
-    update-source-version babashka "''${latest_version/v/}"
+    if [ $(update-source-version babashka-unwrapped "''${latest_version/v/}" --print-changes) = "[]" ]; then
```

Comment:
Ah, I should probably wrap that `$(...)` expression in double quotes, too. Will do with the next round of addressing review feedback!

Reply:
I tested the script locally and it works :) But you are right, better to wrap the expression